frontend/src/components/util/Toggle/toggle.module.scss

```diff
@@ -3,6 +3,10 @@
   padding-left: 4rem;
 }
 
+// .mobile-toggle{
```

Comment:
can we please remove it if it's not required

Reply:
Removed it